config/Dockerfiles/tests.d/aws/02_aws_ec2_vpc_net

```diff
@@ -1,9 +1,8 @@
 #!/bin/bash -xe
 
 # Verify the aws ec2 vpc net provisioning
-# distros.exclude: centos7 fedora28 fedora30
+# distros.exclude: centos7 fedora28 fedora29 fedora30
```

Comment:
Nit: Can we remove fedora28 from all `distros.exclude` and `distros.include`?

Reply:
> Wouldn't it make sense to run most of the tests on Centos since it doesn't change as much as fedora?
makes sense